docs/workload-development-kit/fabric-data-plane.md

```diff
@@ -12,41 +12,98 @@ ms.date: 05/21/2024
+        WorkspaceId: Guid
+        Properties {
+            ConnectionInfo: string
+            CreatedDate: DateTime
+            LastUpdatedTime: DateTime
+        }
+    }
+```
+
+Here, the "ConnectionInfo" property is the FQDN (Fully Qualified Domain Name) of the Warehouse SQL Server. With this FQDN, we'll be able to establish a SQL connection. For more information, see [Connectivity to Data Warehousing in Microsoft Fabric](../data-warehouse/connectivity.md).
+For implementation examples, refer to the [Microsoft Fabric Workload Development Kit](./index.yml).
 
 ### Reading data
 
 Once you authenticate, you can connect to OneLake using [Azure Data Lake Storage REST APIs](/rest/api/storageservices/data-lake-storage-gen2) to read different types of data. We recommend utilizing the [Delta Lake protocol](https://github.com/delta-io/delta/blob/master/PROTOCOL.md) in order to read tables.
 
+Alternatively, if you choose to utilize Azure SQL Database you can implement the following instructions in order to read data from a Warehouse.
+
+1. Create an authorization context. For an example of creating an authorization context, see the [AuthenticateDataPlaneCall method](https://github.com/microsoft/Microsoft-Fabric-workload-development-sample/blob/main/Backend/src/Services/AuthenticationService.cs).
+1. Acquire a token with the Warehouse.Read.All scope on behalf of the user using the bearer token passed from the Frontend.
+1. Using the "Fabric" token, call the "Get Warehouse" API that was previously referenced.
+    -This is required in order to access the Connection info and the display name of the Warehouse, which is the initial catalog of the server.
+1. Acquire a token with SQL scopes on behalf of the user. Again, in order to successfully establish a SQLConnection, you must use the scope https://database.windows.net//user_impersonation.
+1. Use the SQL token and connection information to open a SQL Connection:
+
+    ```csharp
+    private SqlConnection GetSqlConnection(string token, string databaseFqdn, string initialCatalog)
+            {
+                var builder = new SqlConnectionStringBuilder();
+                builder.DataSource = databaseFqdn; // ConnectionInfo from Get Warehouse API
+                builder.InitialCatalog = initialCatalog; //display name of the Warehouse
+                builder.ConnectTimeout = 120;
+    
+                var connection = new SqlConnection();
+                connection.AccessToken = token; // SQL token acquired with the Azure SQL Database user_impersonation scope
+                connection.ConnectionString = builder.ConnectionString;
+                connection.Open();
+                return connection;
+            }
+    ```
+
+1. This connection can now be queried in order to access data from the Warehouse. For more information on utilizing the Microsoft.Data.SqlClient namespace, reference the [Microsoft.Data.SqlClient Namespace Documentation](/dotnet/api/microsoft.data.sqlclient).
```

Comment:
'Microsoft.Data.SqlClient' --> '*Microsoft.Data.SqlClient*'.

'reference the' --> 'see'.